### PR TITLE
Add custom datasource possibilities for postgres_exporter

### PR DESCRIPTION
**Pull request by romdav00**
Hi @bastelfreak , it seems better now :-)

### CODE REVIEW

File: manifests/postgres_exporter.pp [hunk 1]
```diff
@@ -100,24 +104,25 @@
...
-  Boolean $manage_group          = true,
-  Boolean $manage_service        = true,
-  Boolean $manage_user           = true,
-  String[1] $os                  = $prometheus::os,
-  String $options                = '',
-  Optional[String] $download_url = undef,
-  String[1] $arch                = $prometheus::real_arch,
-  String[1] $bin_dir             = $prometheus::bin_dir,
+  Enum['custom', 'env', 'file'] $postgres_auth_method,
+  Hash[String,String] $data_source_custom = {},
```
Comment (bastelfreak) [inline manifests/postgres_exporter.pp]: Can you please enforce the minimal string length?
```suggestion
  Hash[String[1],String[1]] $data_source_custom = {},
```

**Comment by bastelfreak** [comment]
Hi @romdav00, thanks for the PR! Can you have a look at the used email address in the commits? It isn't associated with your github account.

**Comment by bastelfreak** [comment]
Thanks @romdav00. Can you take a look at the first commits in this branch? The used email address isn't associated with your github account.

**Comment by romdav00** [comment]
Hi @bastelfreak is it ok for you ?

**Comment by bastelfreak** [comment]
ah sorry, this PR slipped through. Thanks for pinging!

**Pull request merged**